### PR TITLE
Consolidate config and logging docs

### PR DESCRIPTION
**Pull request by dmitry-s**
Part of #270

### CODE REVIEW

File: spring-cloud-gcp-docs/src/main/asciidoc/config.adoc [hunk 1]
```diff
@@ -95,34 +95,29 @@ above `SampleConfig` bean.
...
+Cloud] provides support to have configuration parameters be
+reloadable with the POST request to `/actuator/refresh` endpoint.
 
-To achieve that, add the Spring Boot Actuator dependency.
-
-Maven coordinates:
-
-----
-<dependency>
+1.  Add the following dependency to your `pom.xml`:
```
Comment (meltsufin) [inline spring-cloud-gcp-docs/src/main/asciidoc/config.adoc]: I would still say "... add the Spring Boot Actuator dependency"

File: spring-cloud-gcp-docs/src/main/asciidoc/config.adoc [hunk 2]
```diff
@@ -95,34 +95,29 @@ above `SampleConfig` bean.
...
-value of the `queueSize` field has been updated.
-
-NOTE: If you're developing locally or just not using authentication in your application, you should
-add `management.security.enabled=false` to your `application.properties` file to allow unrestricted
-access to the `/refresh` endpoint.
+  </dependency>
+....
+2.  Add `@RefreshScope` to your Spring configuration class to have parameters be reloadable at runtime.
+3.  If your application doesn't require authentication, add `management.endpoints.web.expose=*`
+to your `application.properties` at allow unrestricted access to `/actuator/refresh`.
```
Comment (meltsufin) [inline spring-cloud-gcp-docs/src/main/asciidoc/config.adoc]: "at allow"?

File: spring-cloud-gcp-docs/src/main/asciidoc/config.adoc [hunk 3]
```diff
@@ -95,34 +95,29 @@ above `SampleConfig` bean.
...
-configuration and hit the `/refresh` endpoint of your application, you can verify that the
-value of the `queueSize` field has been updated.
-
-NOTE: If you're developing locally or just not using authentication in your application, you should
-add `management.security.enabled=false` to your `application.properties` file to allow unrestricted
-access to the `/refresh` endpoint.
+  </dependency>
+....
+2.  Add `@RefreshScope` to your Spring configuration class to have parameters be reloadable at runtime.
+3.  If your application doesn't require authentication, add `management.endpoints.web.expose=*`
```
Comment (meltsufin) [inline spring-cloud-gcp-docs/src/main/asciidoc/config.adoc]: We probably don't want to recommend `management.endpoints.web.expose=*` as it's too permissive.
See: https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-endpoints.html#production-ready-endpoints-enabling-endpoints

Also, according to current documentation and our sample app, that property is incorrect.
The correct property is `management.endpoints.web.exposure.include`, and I think it's `management.endpoints.web.exposure.include=env` is what we need. Please test with `spring-cloud-gcp-config-sample`.

File: spring-cloud-gcp-docs/src/main/asciidoc/config.adoc [hunk 4]
```diff
@@ -95,34 +95,29 @@ above `SampleConfig` bean.
...
-
-Add a `@RefreshScope` annotation to your class(es) containing remote configuration properties.
-Then, if you change the value of the `myapp.queue-size` variable in the `myapp_prod`
-configuration and hit the `/refresh` endpoint of your application, you can verify that the
-value of the `queueSize` field has been updated.
-
-NOTE: If you're developing locally or just not using authentication in your application, you should
-add `management.security.enabled=false` to your `application.properties` file to allow unrestricted
-access to the `/refresh` endpoint.
+  </dependency>
```
Comment (joaoandremartins) [inline spring-cloud-gcp-docs/src/main/asciidoc/config.adoc]: Why getting rid of Gradle coordinates?
Don't we equally love Gradle? :cry:

File: spring-cloud-gcp-docs/src/main/asciidoc/logging.adoc [hunk 5]
```diff
@@ -37,14 +37,16 @@ For use in Web MVC-based applications, `TraceIdLoggingWebMvcInterceptor` is prov
 
 WARNING: If Spring Cloud GCP Trace is enabled, the logging module disables itself and delegates log correlation to Spring Cloud Sleuth.
 
-There are implementations provided for `TraceIdExtractor`:
+The behavior of the trace ID extraction can be configured with the `spring.cloud.gcp.logging.traceIdExtractor`
```
Comment (joaoandremartins) [inline spring-cloud-gcp-docs/src/main/asciidoc/logging.adoc]: Add `property` at the end?

File: spring-cloud-gcp-docs/src/main/asciidoc/config.adoc [hunk 6]
```diff
@@ -95,34 +95,29 @@ above `SampleConfig` bean.
...
     <artifactId>spring-boot-starter-actuator</artifactId>
-</dependency>
-----
-
-Gradle coordinates:
-
-[source,subs="normal"]
-----
-dependencies {
-    compile group: 'org.springframework.boot', name: 'spring-boot-starter-actuator'
```
Comment (meltsufin) [inline spring-cloud-gcp-docs/src/main/asciidoc/config.adoc]: Let's keep the Gradle coordinates.

**Pull request merged**